src/app/projects/project-list/project-list.component.ts

```diff
@@ -61,11 +66,6 @@ export class ProjectListComponent implements OnInit {
     this.replaceProject(event.projectId);
   }
 
-  viewFavoritesTabIfAny() {
-    if (this.localStorageProjectsService.getFavoriteProjectsIds().length > 0) {
-      this.activeTabIndex = 1;
-    }
-  }
```

Comment:
I think it would be nice to still keep the `viewFavoritesTabIfAny` method and use that as a fallback if there is no query parameter for the tab.